WebApplication/ClientApp/src/components/FetchGoals.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react';
+import App from '../App';
+import { Goal } from './models/Goal';
+
+type State = {
+    goals: string;
+    loading: boolean;
+};
```

Comment:
This type could be used in the 17'th line when defining state property.

Reply:
I don't understand what you mean and how that would be better